lib/main_security.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader of security tests
+# Maintainer: QE Security <none@suse.de>
+
+package main_security;
+use Mojo::Base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use testapi qw(get_var);
+use Utils::Architectures;
+use Utils::Backends;
+
+our @EXPORT = qw(
+  is_security_test
+  load_security_tests
+);
+
+sub is_security_test {
+    return get_var('SECURITY', 1);
```

Comment:
Shouldn't this be `check_var`? or just simple `get_var('SECURITY')` ?

Reply:
check_var might not be ideal as SECURITY might be undef and we end up in comparision against undef variable